src/screens/Followings/elements/TabBar/FollowingTabBar.js

```diff
@@ -75,7 +75,7 @@ const S = StyleSheet.create({
   },
 
   viewborderbottom: {
-    borderBottomColor: colors.holytosca,
+    borderBottomColor: COLORS.holyTosca,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color reference `colors.holytosca` has been replaced with `COLORS.holyTosca`. This change is consistent with the new naming convention and improves maintainability.
```diff
-    borderBottomColor: colors.holytosca,
+    borderBottomColor: COLORS.holyTosca,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->